workflow/hpxml_inputs.json

```
@@ -842,6 +842,11 @@
   "sample_files/base-bldgtype-mf-unit-adjacent-to-other-heated-space.xml": {
     "parent_hpxml": "sample_files/base-bldgtype-mf-unit.xml"
   },
+  "sample_files/base-bldgtype-mf-unit-adjacent-to-other-heated-space-hvac-none.xml": {
+    "parent_hpxml": "sample_files/base-bldgtype-mf-unit-adjacent-to-other-heated-space.xml",
```

We should test all the MF space types:
```suggestion
  "sample_files/base-bldgtype-mf-unit-adjacent-to-multiple-hvac-none.xml": {
    "parent_hpxml": "sample_files/base-bldgtype-mf-unit-adjacent-to-multiple.xml",
```